src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Install and configure the Prometheus agent alongside the Kubernetes integration
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus agent
+metaDescription: 'How to install, update, or uninstall your New Relic Prometheus agent alongside the Kubernetes integration.'
+---
+
+import infrastructurePrometheusIoDashboard from 'images/infrastructure_screenshot-crop_prometheus-io-dashboard.png'
+
+import infrastructurePrometheusNrqlQuery from 'images/infrastructure_screenshot-crop_prometheus-nrql-query.png'
+
+New Relic's Prometheus agent is a thin wrapper on top of Prometheus server to run it in agent mode. With this integration, you can create a YAML configuration file used by Prometheus that's ready to send metrics to New Relic.
+
+Learn more about the solution from [its repository](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/charts/newrelic-prometheus-agent/values.yaml#L184).
+
+## Install the Prometheus agent alongside the Kubernetes integration [#kubernetes-install]
+
+You can install our [Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration) to get a full observability of your Kubernetes clusters, which includes the Prometheus agent.
+
+1. Install our Kubernetes integration. See how to install it [here](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure).
+
+<Callout variant="tip">
+  Alternatively, we also offer fully manual instructions for [deploying our integration using Helm](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm).
+</Callout>
+
+2. To ensure the integration has been configured correctly, go to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > Query builder**, and run this NRQL query to see if data has been reported:
+
+  ```sql
+  FROM Metric SELECT count(*) WHERE collector.name = 'prometheus-agent' AND cluster_name = 'YOUR_CLUSTER_NAME' since 1 hour ago
+  ```
+
+<Callout variant="tip">
+  If you don't see your data right away, wait for a few seconds. Data may take some time to make it to New Relic.
+</Callout>
+
+By default, the New Relic Prometheus agent uses two annotations to discover targets: `newrelic.io/scrape: "true"` and `prometheus.io/scrape: "true"`. All the targets in the cluster that are annotated with `newrelic.io/scrape: "true"` are discovered and scraped. However, the default annotation from the Prometheus community, `prometheus.io/scrape: "true"`, is discovered but we filter them to only scrape the [Prometheus integrations](/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro) that we support.
```

Comment:
We are in the `## Install the Prometheus agent alongside the Kubernetes integration [#kubernetes-install]` section, why are we starting to talk about the New Relic Prometheus agent default behaviour?

Reply:
I restructured the document. Now the default mode is mentioned in a separate section (Define which endpoints to scrape")
Should we add more information about how to manually modify the jobs?